R/benchmark.R

```diff
@@ -93,6 +98,8 @@ Benchmark <- function(name,
                       after_each = TRUE,
                       teardown = TRUE,
                       valid_params = function(params) params,
+                      case_version = function(params) NULL,
+                      packages_used = function(params) "arrow",
```

Comment:
Added this because it's already called, but previously we were relying on every benchmark adding it in the same way though `...`. I'm now leaning more towards R6-ifying this class